tests/tracing.spec.ts

```diff
@@ -139,6 +139,8 @@ test('should not include trace resources from the provious chunks', async ({ con
   await page.goto(server.EMPTY_PAGE);
   await page.setContent('<button>Click</button>');
   await page.click('"Click"');
+  // Give it enough time for both screenshots to get into the trace.
+  await new Promise(f => setTimeout(f, 1000));
```

Comment:
what about our users?

Reply:
We should probably address that but it's not the subject of this test which is just checking that the artifacts are properly isolated between chunks.